packages/eslint-config/plugins/node-mjs.js

```diff
@@ -0,0 +1,16 @@
+// plugin:node/recommended: https://github.com/mysticatea/eslint-plugin-node/blob/master/lib/index.js
+module.exports = {
+  // considers all files as ES Modules.
+  extends: ['plugin:node/recommended-module'],
+  rules: {
+    'node/no-unpublished-require': 'off',
+  },
+  // some config file also use commonjs module
+  globals: {
+    __dirname: 'readonly',
```

Comment:
`false` is shorter than `'readonly'`. Of course it is not so obvious.

Reply:
The use of older values is deprecated, eslint recommended this way.

> For each global variable key, set the corresponding value equal to "writable" to allow the variable to be overwritten or "readonly" to disallow overwriting.
> For historical reasons, the boolean value false and the string value "readable" are equivalent to "readonly". Similarly, the boolean value true and the string value "writeable" are equivalent to "writable". However, the use of older values is deprecated.

https://eslint.org/docs/user-guide/configuring#specifying-globals